extensions/adobe/experience/adcloud/dsp/package.schema.json

```diff
@@ -0,0 +1,104 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/dsp/package",
```

Comment:
Yes, there is also  'package_goal' table involved , but not promo2.goal_interval, promo2.goal_metric and promo2.goal_type tables. I added all these extra fields because they seemed relevant in the context of a package entity. Do you agree ? I've documented it [here](https://wiki.corp.adobe.com/display/EfficientFrontier/%5BXDM+Mapping%5D+Campaign+Placement-related+hierarchy#id-[XDMMapping]CampaignPlacement-relatedhierarchy-Package):